packages/blade/src/components/Card/_decisions/interactive-cards.md

```diff
@@ -0,0 +1,251 @@
+    <Card isSelected={selected === 'monthly'}>
+      <CardBody>
+        <Text>Monthly</Text>
+      </CardBody>
+    </Card>
+  </CardGroupItem>
+  <CardGroupItem value="quarterly">
+    <Card isSelected={selected === 'quarterly'}>
+      <CardBody>
+        <Text>Quarterly</Text>
+      </CardBody>
+    </Card>
+  </CardGroupItem>
+  <CardGroupItem value="annually">
+    <Card isSelected={selected === 'annually'}>
+      <CardBody>
+        <Text>Annually</Text>
+      </CardBody>
+    </Card>
+  </CardGroupItem>
+</CardGroup>;
+```
+
+**Pros:**
+
+- No overhead on consumer end, and the API will be intuitive to exisiting radio/checkbox components
+- Will work on both web & native seamlessly, and we can ensure a11y too
+
+**Cons:**
+
+- Bit of effort on our end, we will basically have to create subset of Radio/Checkbox components for Card's usecase
+- Introduces new components, CardGroup, CardGroupItem.
+
+**My Opinion:** Method 2 will be much more consistent across platforms and neater than letting consumer hack around various combinations of tricks to get single/multi selection to work.
+
+
+## Linkable Card
+
+With linkable card, the whole card itself can be clicked or linked to any URL. 
+
+With linkable card the main issue is accessibility & interaction, because simpliy wrapping the whole card in an `a` tag won't be valid, because inside the card there can also be interactive elements.
+
+Three problems with naive approach of just wrapping the card in a link: 
+
+- The wrapped Link component's styling will be leaked into the internal interactive components, see how accordion items and the normal Text component is underlined
+- If we have any interactive component say, Accordion or Button clicking on those will also trigger the Link navigation.
+- If we open screen reader and try to focus on the card, it will announce the whole card's content as the text of the link, ideally it should only announce the action or label text of the link.
+
+> Check this [Codesandbox for reproduction](https://codesandbox.io/s/card-nested-links-issue-j6wtyn?file=/App.tsx)
+
+
+**Solution:**
+
+We need to overlay the Link to cover the card instead of nesting it.
+- ChakraUI uses [LinkOverlay](https://chakra-ui.com/docs/components/link-overlay/usage#nested-links) component to solve this
+- MUI uses a `overlay` prop on the [link](https://mui.com/joy-ui/react-card/#multiple-actions) itself
+
+Previously [discussed](https://razorpay.slack.com/archives/G01B3LQ9H0W/p1689580390134439) this with chaitanya:
+
+Can explore a solution where we don't need to expose `overlay` prop or `LinkOverlay` component to consumer by accepting `<Card href="" />` but internally we will anyway have a `LinkOverlay` component for reasons mentioned above.
+
+Example usage: 
+
+```jsx
+// we might also need to expose other link props like target=blank etc
+<Card href="/payment-pages" />
+```
+
+
+
+## Open Questions
+
+- What about client-side nagivation with linkable cards?
```

Comment:
Add decision that this will be prioritised when we solve for client side navigation on our `Link` component, currently will be scoped out.